backend/curation/api/v1/curation/collections/common.py

```diff
@@ -208,20 +202,16 @@ def reshape_dataset_for_curation_api_common(
     return ds
 
 
-def reshape_dataset_for_curation_api(dataset_version: DatasetVersion, use_canonical_url: bool, preview=False) -> dict:
-    dataset = reshape_dataset_for_curation_api_common(dataset_version, use_canonical_url, preview)
+def reshape_dataset_for_curation_api_as_canonical(
```

Comment:
nit: suggestion but not a required change--I think it would be more consistent with the pattern we've been using in other functions in this class if we turned 'as_canonical' into an optional flag in `reshape_dataset_for_curation_api` (ala `as_version` in the `reshape_collection_for_curation_api` function) and added the below published_at & revised_at logic there, instead of the separate function

Reply:
done